### ISSUE TITLE
Hide in menubar

### ISSUE THREAD

**Issue by Erik262** [dexterleng/vimac#132]
Hello, 

I would appreciate to be able, to select the visibility of the icon on the menu-bar in the preferences. Since every App is showing there and it looks kind of messy.

Nice work by the way!

**Comment by dsernst** [comment]
You might be interested in this other MacOS utility: Hidden Bar https://github.com/dwarvesf/hidden


![](https://github.com/dwarvesf/hidden/raw/develop/img/tutorial.gif)

**Comment by Erik262** [comment]
This might be a solution for having many apps, but I want only hide the vimac one.

**Comment by muyinliu** [comment]
> You might be interested in this other MacOS utility: Hidden Bar https://github.com/dwarvesf/hidden
> 
> ![](https://github.com/dwarvesf/hidden/raw/develop/img/tutorial.gif)

See also: [Dozer](https://github.com/Mortennn/Dozer) (seems better than `hidden`)

**Comment by dexterleng** [comment]
Good Idea to have this option, my menu bar is really full with all these menu bar apps :).

**Comment by skobylite** [comment]
you’re really going full tilt with vimac, absolutely love seeing you turn this from idea to reality over the last year. I also use Hidden bar but would also vote for an option to hide the menu bar icon separately

**Comment by nickpodratz** [comment]
Agree! Would be really nice to just hide the icon by (un)ticking some box in the settings.
